handlers/handlers_test.go

```diff
@@ -75,10 +73,11 @@ func TestUnitHandlers(t *testing.T) {
 
 		Convey("test CreateFilterID returns 500 if unable to create filter job on filter api", func() {
 			mockClient := NewMockFilterClient(mockCtrl)
-			mockClient.EXPECT().CreateJob(gomock.Any()).Return("", errors.New("no filter job for you"))
+			mockClient.EXPECT().CreateJob(gomock.Any(), gomock.Any()).Return("", errors.New("no filter job for you"))
```

Comment:
There shouldn't be any reference to filter job, there is only filter blueprints and filter outputs.